webview/src/shared/components/sectionContainer/InfoTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import cx from 'classnames'
+import { PlotsSection } from 'dvc/src/plots/webview/contract'
+import { SetupSection } from 'dvc/src/setup/webview/contract'
+import styles from './styles.module.scss'
+import { SectionDescription } from './SectionContainer'
+import Tooltip from '../tooltip/Tooltip'
+import { Icon } from '../Icon'
+import { Info, PassFilled, Error } from '../icons'
+
+export enum TooltipIconType {
+  PASSED = 'icon-pass-filled',
+  INFO = 'icon-info',
+  ERROR = 'icon-error'
+}
+
+const getIcon = (icon?: TooltipIconType) => {
+  const props = {
+    className: styles.infoIcon,
+    'data-testid': icon || TooltipIconType.INFO,
+    height: 16,
```

Comment:
I'd rather see these properties that do not change directly on the component so that I don't have to check the code to see whether they've been changed or not.